security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java

```diff
@@ -286,15 +303,10 @@ public void handleMessage(SoapMessage message) throws Fault {
                                     String xpathStrict = "/Layout/Policy/Strict";
                                     String xpathLaxTimestampFirst = "/Layout/Policy/LaxTimestampFirst";
                                     String xpathLaxTimestampLast = "/Layout/Policy/LaxTimestampLast";
-                                    layoutLax = evaluateExpression(xml, xpathLax);
-                                    layoutStrict = evaluateExpression(xml, xpathStrict);
-                                    layoutLaxTimestampFirst = evaluateExpression(xml, xpathLaxTimestampFirst);
-                                    layoutLaxTimestampLast = evaluateExpression(xml, xpathLaxTimestampLast);
 
```

Comment:
Can we make sure we have a ticket for this and maybe comment out the code that we need to keep for it and put TODO - Ticket #.